tests/e2e/Jenkinsfile

```diff
@@ -1,51 +1,68 @@
 pipeline {
   agent any
   libraries {
-    lib('fxtest@1.9')
+    lib('fxtest@1.10')
   }
   options {
     ansiColor('xterm')
     timestamps()
     timeout(time: 5, unit: 'MINUTES')
   }
-  environment {
-    PYTEST_PROCESSES = "${PYTEST_PROCESSES ?: "auto"}"
-    PYTEST_ADDOPTS =
-      "-n=${PYTEST_PROCESSES} " +
-      "--color=yes"
-    PULSE = credentials('PULSE')
-  }
   stages {
     stage('Lint') {
+      agent {
+        dockerfile {
+          dir 'tests/e2e'
+        }
+      }
       steps {
-        sh "tox -c tests/e2e/tox.ini -e flake8"
+        sh "flake8 tests/e2e/"
```

Comment:
I'm not sure if this will pick up the configuration in tests/e2e/setup.cfg.. could you confirm?

Reply:
I can!:

Fails without ignore=E501 in setup.cfg, here: https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/go-bouncer.adhoc/75/console, from https://github.com/mozilla-services/go-bouncer/pull/203/commits/c5c8c19e886eafbc305f7bcfeaf1623ce68f4aea.

PASSes when restoring ignore=501 in setup.cfg in commit https://github.com/mozilla-services/go-bouncer/pull/203/commits/ac68029c2a951caa684fb10ea88249a29e16bbce:

https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/go-bouncer.adhoc/76/console